<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>413</width>
    <height>284</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Saving and Loading</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="ckbSaveFile">
            <property name="text">
             <string>Auto save file before exit</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="ckbOpenLastFiles">
            <property name="text">
             <string>Open last opened files on startup</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Gui</string>
         </property>
         <widget class="QCheckBox" name="ckbHideMenuBar">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>91</y>
            <width>282</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Hide menu bar(use Alt or F10 key to toggle)</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="ckbHideStatusBar">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>110</y>
            <width>114</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Hide status bar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_31">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>20</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Style:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QComboBox" name="cmbStyle">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>18</y>
            <width>171</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>46</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>&amp;Language:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>cmbLanguage</cstring>
          </property>
         </widget>
         <widget class="QComboBox" name="cmbLanguage">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>46</y>
            <width>171</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QRadioButton" name="rbLarge">
          <property name="geometry">
           <rect>
            <x>253</x>
            <y>70</y>
            <width>53</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Large</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="rbMedium">
          <property name="geometry">
           <rect>
            <x>189</x>
            <y>70</y>
            <width>59</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Medium</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>70</y>
            <width>114</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Tool bar icon size:</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="rbSmall">
          <property name="geometry">
           <rect>
            <x>131</x>
            <y>70</y>
            <width>53</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Small</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabShortcut">
      <attribute name="title">
       <string>Shortcut</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="leFilter">
            <property name="placeholderText">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TShortcutView" name="tvShortcut">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Shortcut:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QKeySequenceEdit" name="ksShortcut"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnShortcutSet">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Apply shorcut change to command.</string>
              </property>
              <property name="text">
               <string>Set</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/other/images/ok_set.png</normaloff>:/other/images/ok_set.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Debug</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Engine path:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leEnginePath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>361</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOpenDir">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>28</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/chreditor.qrc">
         <normaloff>:/other/images/open2.png</normaloff>:/other/images/open2.png</iconset>
       </property>
      </widget>
      <widget class="QCheckBox" name="ckbMultiInstances">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Check to enable running multiple instances of engine.</string>
       </property>
       <property name="text">
        <string>Multiple instances.</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Author</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Author:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leAuthor">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Input default author name.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Contact:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leContact">
         <property name="placeholderText">
          <string>Input default contact information here.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <widget class="QCheckBox" name="ckbAutoLocate">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>291</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Automatic locate image in scaling scene.</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TShortcutView</class>
   <extends>QTreeView</extends>
   <header location="global">shortcutview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ckbSaveFile</tabstop>
  <tabstop>ckbOpenLastFiles</tabstop>
 </tabstops>
 <resources>
  <include location="../resource/chreditor.qrc"/>
 </resources>
 <connections/>
</ui>
